api/v1/openapi.yaml

```
@@ -1778,6 +1778,9 @@ definitions:
       encryption:
         description: Status of transparent encryption
         "$ref": "#/definitions/EncryptionStatus"
+      cni-chaining:
+        description: Status of CNI chaining encryption
```

I assume copy-pasta :smile: 

```suggestion
        description: Status of CNI chaining
```

---

you got me :(, Ctrl C + Ctr V are my fav keys :face_exhaling: